service/controller/orgpermissions/resource/membership/create.go

```diff
@@ -2,99 +2,16 @@ package membership
 
 import (
 	"context"
-	"fmt"
-	"reflect"
-
-	"github.com/giantswarm/microerror"
-	rbacv1 "k8s.io/api/rbac/v1"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	pkgkey "github.com/giantswarm/rbac-operator/pkg/key"
-	"github.com/giantswarm/rbac-operator/pkg/label"
-	"github.com/giantswarm/rbac-operator/pkg/project"
-	"github.com/giantswarm/rbac-operator/service/controller/orgpermissions/key"
 )
 
 // Ensures that a ClusterRoleBinding '<rolebinding-name>-organization-<organization>-read'
 // exists for a certain RoleBinding, with the same subjects.
+//
+// DEPRECATED
+// This resource is replaced by the rbac/externalresources resource.
+// The deletion logic remains to help cleaning up
+//
 func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
-	var err error
-
-	roleBinding, err := key.ToRoleBinding(obj)
-	if err != nil {
-		return microerror.Mask(err)
-	}
-
-	if !pkgkey.IsOrgNamespace(roleBinding.Namespace) || !isTargetRoleBinding(roleBinding) {
-		return nil
-	}
-
-	orgName := pkgkey.OrganizationName(roleBinding.Namespace)
-	orgReadClusterRoleBindingName := pkgkey.OrganizationReadClusterRoleBindingName(roleBinding.Name, orgName)
-	orgReadClusterRoleName := pkgkey.OrganizationReadClusterRoleName(roleBinding.Namespace)
-
-	orgReadClusterRoleBinding := &rbacv1.ClusterRoleBinding{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "ClusterRoleBinding",
-			APIVersion: "rbac.authorization.k8s.io/v1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name: orgReadClusterRoleBindingName,
-			Labels: map[string]string{
-				label.ManagedBy: project.Name(),
-			},
-		},
-		Subjects: roleBinding.Subjects,
-		RoleRef: rbacv1.RoleRef{
-			APIGroup: "rbac.authorization.k8s.io",
-			Kind:     "ClusterRole",
-			Name:     orgReadClusterRoleName,
-		},
-	}
-	err = r.createOrUpdateClusterRoleBinding(ctx, orgReadClusterRoleBinding)
-	if err != nil {
-		return microerror.Mask(err)
-	}
+	// return r.EnsureDeleted(ctx, obj)
```

Comment:
Not sure if this will work but I would like to try after we made sure that the new logic works